### PR TITLE
yubikey-oath-dmenu: Customize dmenu prompt

### LINKED ISSUE #7: Allow custom dmenu prompt
Currently, the dmenu prompt is hardcoded to "Credentials". Could be nice to have a way to customise that.

### PR DESCRIPTION
**Pull request by agherzan**
Fixes #7

Signed-off-by: Andrei Gherzan <andrei@gherzan.ro>

### CODE REVIEW

File: yubikey-oath-dmenu [hunk 1]
```diff
@@ -56,7 +56,7 @@ def touch_callback(ctx):
 @click.option('--clipboard', metavar='CLIPBOARD',
               help='Passed through as -selection option to xclip')
 @click.option('--dmenu-prompt', 'dmenu_prompt', metavar='PROMPT', default = 'Credentials:',
-              help='Passed as --prompt argument to dmenu')
```
Comment (agherzan) [inline yubikey-oath-dmenu]: Right.

**Review by emlun — APPROVED** [review]
Great, thanks!

**Comment by emlun** [comment]
This is now in master, but I don't think I'll have time to produce a proper release until at least a few days from now.

**Comment by agherzan** [comment]
No need for a release soon. It's good whenever. I can deal with "credentials" for a while :)

**Comment by emlun** [comment]
Hey, I just discovered this feature was already supported without the `--dmenu-prompt` option - turns out additional `-p` arguments override the hard-coded one. The argument passthrough feature was added a mere 2 days before this PR and its issue were opened, though, so I guess it slipped us both by.

With that, would you mind if I remove the `--dmenu-prompt` option?

**Comment by agherzan** [comment]
@emlun if you tested that I'm fine. It was surely not the case when I pushed this PR.

**Pull request merged**